### PR TITLE
remote productivity.md added

### PR DESCRIPTION
**Pull request by KBanecka**
Signed-off-by: Kamila Banecka <kamila.banecka@3mdeb.com>

### CODE REVIEW

File: blog/content/post/2020-04-16-remote_work_post.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+---
+title: Remote productivity
+abstract: 'About remote productivity in 3mdeb.'
```
Comment (pietrushnic) [inline blog/content/post/2020-04-16-remote_work_post.md:3]: @KBanecka this should be something that post as pitch on social media, it seems to be little bit too short and not very encouraging.

File: blog/content/post/2020-04-16-remote_work_post.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+the fresh air or doing sports. Responding to encountered difficulties, we adapt
+to the current situation, maintaining the good quality of our work. All team
+agrees that it will definitely be a positive impact. The current situation
+teaches us better work organization, self-discipline and strengthens the sense
+of responsibility. It will be undeniably easier for us to stay focused in the
+work environment with the colleagues sitting nearby. After such a break, it will
+be much easier for us to appreciate the office everyday. What we will miss about
+remote working? Surely the flexibility of working hours and the lack of
+commuting. Nevertheless, we are all waiting for the end of the pandemic and
+quick return to our desks, screens and mugs.
```
Comment (pietrushnic) [inline blog/content/post/2020-04-16-remote_work_post.md:66]: @KBanecka I like the social feeling and emotion of the post, but we are open-source software and hardware company as well as embedded systems consulting. I would appreciate if you could underline how our beliefs, experience, past project and open technologies that we using every day helping us in remote work productivity.

For example, this is a very good post from well-respected company: https://puri.sm/posts/our-essential-list-of-free-software-for-remote-work/

File: blog/content/post/2020-04-16-remote_work_post.md (hunk 1 above)
Comment (KBanecka) [reply blog/content/post/2020-04-16-remote_work_post.md:3]: Ok, I'll work it out

File: blog/content/post/2020-04-16-remote_work_post.md (hunk 2 above)
Comment (KBanecka) [reply blog/content/post/2020-04-16-remote_work_post.md:66]: @pietrushnic I'm including your suggestions

File: blog/content/post/2020-04-16-remote_work_post.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+Since the recent COVID-19 outbreak we all have been forced to pack our tools
+along with our potential and switch to home office. One may say that for IT it
+is not a milestone; after all our work didn’t change significantly. We can still
+develop solutions of an invariably good quality providing that our remote
+environment has been fully set. Apart from tools there are more factors,
+combined in workflow, of which we are deprived while working remotely for a long
+term. Among endless interruptions and no watchful eyes around, with difficulties
+to connect people directly, there is the question that rises inside the head:
+<b> Ping or not to ping? </b>
```
Comment (artur-rs) [inline blog/content/post/2020-04-16-remote_work_post.md:28]: @KBanecka html elements are necessary evil in this case - please use markdown emphasis characters:

\*\*bold\*\* -> **bold**

or

\_\_bold\_\_ -> __bold__

File: blog/content/post/2020-04-16-remote_work_post.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
+---
+title: Remote productivity
```
Comment (artur-rs) [inline blog/content/post/2020-04-16-remote_work_post.md:2]: @KBanecka check this tool: https://aminstitute.com/headline/

**Comment by artur-rs** [comment]
@KBanecka do not forget about creating new profile page according to this guide: https://github.com/3mdeb/news-and-ideas#add-new-profile-page

File: blog/content/post/2020-04-16-remote_work_post.md (hunk 1 above)
Comment (KBanecka) [reply blog/content/post/2020-04-16-remote_work_post.md]: [commit](https://github.com/3mdeb/news-and-ideas/commit/dd554d2ecb716dae3e9e9afe9585bc912cd927c1#diff-a68390b5dcec9ebb7b7e5492bfbe6cdc)

File: blog/content/post/2020-04-16-remote_work_post.md (hunk 2 above)
Comment (KBanecka) [reply blog/content/post/2020-04-16-remote_work_post.md]: [commit](https://github.com/3mdeb/news-and-ideas/commit/dd554d2ecb716dae3e9e9afe9585bc912cd927c1#diff-a68390b5dcec9ebb7b7e5492bfbe6cdc)

File: blog/content/post/2020-04-16-remote_work_post.md [hunk 5]
```diff
@@ -0,0 +1,101 @@
+---
+title: Tools and ideas that keep us behind the wheel while remote working.
```
Comment (pietrushnic) [inline blog/content/post/2020-04-16-remote_work_post.md:2]:
```suggestion
title: Tools and ideas that keep us behind the wheel while working remotely.
```
?

**Comment by artur-rs** [comment]
@KBanecka `.json` file is still missing (check the profile guide again). Also, please launch blog locally and check if post is rendering correctly: https://github.com/3mdeb/news-and-ideas#local-preview

File: blog/content/post/2020-04-16-remote_work_post.md [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+ possibility to mute them or to change the environment, even in the form of a
+ short walk. And yes, we do love them more than work (at least some of us) but
+ staying focused for a long time among sounds and needs is simply tough.
+ However, our motivation goes even stronger, as developing open-source solutions
+ to the public good in the current situation becomes a real mission. Majority
+ works remotely these days and we all need to be equipped with the solutions of
+ a good quality and open way to modify them according to our needs. If you want
+ to check our particular contributions, the
+ [article](https://blog.3mdeb.com/2020/2020-04-16-3mdeb-contrib-q1/) is the
+ must-read one.
```
Comment (pietrushnic) [inline blog/content/post/2020-04-16-remote_work_post.md:53]: @KBanecka why we have spece at the beginning of every line!?

File: blog/content/post/2020-04-16-remote_work_post.md [hunk 7]
```diff
@@ -0,0 +1,101 @@
...
+  agree that our struggles are going to have a positive impact when back to
+  office, as current situation taught us about better work organization,
+  self-discipline and the sense of responsibility. When it will become easier
+  for us to stay focused in the work environment with the colleagues sitting
+  nearby, what we will miss about remote working? Surely the flexibility of
+  working hours, lack of commuting and in some cases homey pyjamas.
+  Nevertheless, we are all waiting for the end of the pandemic and quick return
+  to our desks, screens and mugs.
+
+  ![remote_work02](/img/remote_work02.jpg)
```
Comment (artur-rs) [inline blog/content/post/2020-04-16-remote_work_post.md:101]: @KBanecka there is no summary? Is it intentional?

File: blog/content/post/2020-04-16-remote_work_post.md [hunk 8]
```diff
@@ -0,0 +1,101 @@
...
+ Maintaining full concentration among domestic interruptions is not an only
+wrinkle. Lack of direct contact with co-workers is the obvious issue we all cope
+with recently, as it extends sometimes the performance or full-understanding of
+certain tasks. With a little help comes Jitsi Meet on Docker. As we use Docker
+widely in our projects for firmware and embedded software building, conducting
+trainings and when rootfs building for infrastructure deployment, here again,
+for Jitsi Meet, fully encrypted, 100% Open Source videoconferencing solution,
+you can find in our repository all the necessary tools to run a [Jitsi
+Meet](https://github.com/3mdeb/docker-jitsi-meet) stack on Docker using Docker
+Compose.  Another grind-team-gears issue is difficulty with maintaining proper
```
Comment (pietrushnic) [inline blog/content/post/2020-04-16-remote_work_post.md:64]: @KBanecka new paragraph?

File: blog/content/post/2020-04-16-remote_work_post.md [hunk 9]
```diff
@@ -0,0 +1,101 @@
...
+work-life balance. It is easy to merge with desk when having all the necessities
+for the taking along with the good tools to work on. We didn't investigate
+whether our testers work in pyjamas, but let's have a glimpse on what they are
+having in the basic toolkit: [Flashrom](https://github.com/3mdeb/flashrom),
+utility designed for flashing images on various programmer devices, helpful in
+preparing device for further operations such as testing. Worthy to mention
+[Cukinia](https://github.com/3mdeb/cukinia), Linux firmware validation framework
+helps to run simple system-level validation tests on firmware, and when
+analyzing security, Chipsec with it's security test suite, tools for accessing
```
Comment (pietrushnic) [inline blog/content/post/2020-04-16-remote_work_post.md:73]:
```suggestion
analyzing security, [Chipsec](https://github.com/chipsec/chipsec) with its security test suite, tools for accessing
```

File: blog/content/post/2020-04-16-remote_work_post.md (hunk 3 above)
Comment (KBanecka) [reply blog/content/post/2020-04-16-remote_work_post.md]: changed

File: blog/content/post/2020-04-16-remote_work_post.md (hunk 5 above)
Comment (KBanecka) [reply blog/content/post/2020-04-16-remote_work_post.md:2]: Both ways are ok

File: blog/content/post/2020-04-16-remote_work_post.md (hunk 6 above)
Comment (KBanecka) [reply blog/content/post/2020-04-16-remote_work_post.md:53]: changed

File: blog/content/post/2020-04-16-remote_work_post.md (hunk 3 above)
Comment (KBanecka) [reply blog/content/post/2020-04-16-remote_work_post.md]: done

**Pull request merged**